proxy/director.go

```diff
@@ -126,7 +126,7 @@ func timedUnavailabilityFunc(wait time.Duration) func(*endpoint) {
 	return func(ep *endpoint) {
 		time.AfterFunc(wait, func() {
 			ep.Available = true
-			log.Printf("proxy: marked endpoint %s available", ep.URL.String())
+			log.Printf("proxy: marked endpoint %s available to retry it until refresh interval time arrived.", ep.URL.String())
```

Comment:
could we improve this message? I cannot understand it well still.

Maybe: ... after refresh interval to hope the endpoint back online